eslint.config.mjs

```diff
@@ -0,0 +1,47 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import path from 'node:path';
+import eslint from '@eslint/js';
+import { includeIgnoreFile } from '@eslint/compat';
+import tseslint from 'typescript-eslint';
+import eslintPrettier from 'eslint-plugin-prettier/recommended';
+import unicornPlugin from 'eslint-plugin-unicorn';
+import headerPlugin from 'eslint-plugin-header';
+
+// https://github.com/Stuk/eslint-plugin-header/issues/57
+headerPlugin.rules.header.meta.schema = false;
```

Comment:
See here: https://github.com/Stuk/eslint-plugin-header/issues/57

We depend on an unmaintained plugin. It has compatibility issues with the latest eslint, but still works with this patch.

Works for now, but we need to think about replacement. Maybe it is time to write our own plugin and drop it here: https://github.com/cloudscape-design/build-tools